relay/blockchain/node.py

```diff
@@ -6,18 +6,24 @@
 
 class Node:
 
-    def __init__(self, web3):
+    def __init__(self, web3, is_parity=True):
         self._web3 = web3
+        self.is_parity = is_parity
 
     def relay_tx(self, rawtxn):
         return self._web3.eth.sendRawTransaction(rawtxn)
 
     def transaction_receipt(self, txn_hash):
         return self._web3.eth.getTransactionReceipt(txn_hash)
 
-    def get_tx_infos(self, user_address):
-        return TxInfos(balance=self._web3.eth.getBalance(user_address),
-                       nonce=self._web3.eth.getTransactionCount(user_address),
+    def get_tx_infos(self, user_address, block_identifier='pending'):
+        if self.is_parity and block_identifier == 'pending':
+            nonce = int(self._web3.manager.request_blocking('parity_nextNonce', [user_address]), 16)
+            print('paritynonce')
```

Comment:
this should be removed or turned into a logging call

Reply:
Woops my bad. Of course.